### PR TITLE
Create eip-2794.md

### PR DESCRIPTION
**Pull request by zhous**
Fixing the problem 2 EIPs mixed.

### CODE REVIEW

File: EIPS/eip-2794.md [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+status: Draft
+type: Standards Track
+category: Core
+created: 2020-07-17
+---
+
+## Simple Summary
+We propose a new staking protocol with an honor system to replace ETH 2.0's staking mechanism in which a node must stake 32 ETHs to be a validator and thus expose ETH 2.0 to extremely high risks. This proposal will bring strong and reliable security to ETH 2.0's staking.
+
+## Motivation (Risk Analysis)
```
Comment (MicahZoltu) [inline EIPS/eip-2794.md:15]: Use the template names here, not custom ones.
```suggestion
## Motivation
```

File: EIPS/eip-2794.md [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+
+As a rule of thumb, if ETH's price rises by ten times or has a hike like what happened in 2017, ETH 2.0's validators will be tempted to sell ETHs for fiat currencies or stable coin and it will be a huge financial burden for people who want to join staking. When this happens, on one hand a large number of existing validators will perform an exit, and on the other hand interested people who want to stake may not be able to afford the expensive cost. The consequence will be that ETH 2.0's security and reliability will be hit hardly.
+
+Vitalik Buterin mentioned this kind of risk in a reddit post but he emphasized that although it may happen infrequently in practice it may still happen. However, from our perspective, even if this happens once, it will be a disaster to Ethereum.
+
+Therefore it is necessary to have a solution for this extreme scenario.
+
+## Abstract (Two Strategies)
```
Comment (MicahZoltu) [inline EIPS/eip-2794.md:24]: Use template name:
```suggestion
## Abstract
```

File: EIPS/eip-2794.md [hunk 3]
```diff
@@ -0,0 +1,121 @@
...
+- An honor token can by purchased and sold on exchanges such as OpenSea as easy as with just one click. This can be easily implemented with support from these exchanges.
+- Honor tokens will bring both value and wealth to validators, and will very likely be rarities and collectibles. In our proposal when validators exit they will receive NDAO rather than ETH. In this case if ETH's price is higher than it used to be, validators may feel unfair and honor tokens will be a good compensation.
+- Honor tokens are good incentives to attrace validators to join staking as early as possible because these early birds may win unique token with their service time which may be the longest among all the service times recorded in all honor tokens. This is the first ever means to reward an Ethereum's contributor with special honor and will motivate future thoughts in blockchain's development.
+
+From our perspective, the above measures will eliminate the risks of ETH 2.0 staking being harmed by ETH price fluctuations.
+
+## NDAO and ETH Forging
```
Comment (MicahZoltu) [inline EIPS/eip-2794.md:58]: This is not one of the template sections found in https://raw.githubusercontent.com/ethereum/EIPs/master/eip-template.md.  While technically additional template sections are allowed, they are weakly discouraged.  Recommend including this into one of the template sections.

File: EIPS/eip-2794.md [hunk 4]
```diff
@@ -0,0 +1,121 @@
...
+- The RICP's algorithm runs this way:
+It is initialized with ETH's circulating supply.
+When a trader sends an amount of ETHs to the system, the ETH's circulating supply will be deducted by this amount and the total supply of ETH in the trading pair will be deducted by this amount. We can think of as these ETHs are burned.
+For instance if ETH's total supply is 107,477,373 , a trader sends 100 ETHs to this system in exchange for NDAOs, then the total supply of ETH in the algorithm will be 107,477,273 which will be less than the previous supply by 100, and the ETH's price will rise based on the RICP algorithm.
+- The total supply of ETH in the system and the total supply of NDAO are calculated once on a daily basis.
+- In this system ETH is priced in NDAO.
+
+Note: right after the foring contracts start, NDAO's pegging relationship with USD will be revoked.
+## Improvement to Economic Incentives
```
Comment (MicahZoltu) [inline EIPS/eip-2794.md:82]: Not a template section.  See https://raw.githubusercontent.com/ethereum/EIPs/master/eip-template.md

File: EIPS/eip-2794.md [hunk 5]
```diff
@@ -0,0 +1,121 @@
...
+- Our proposal can reward validators on a daily basis. This is more attractive to validators who can only get their rewards after Phase 1 in the current mechanism.
+- ETH's inflation rate will become lower.
+- NDAO will be in great demand. Finally validators will need a huge amount of NDAOs for staking. And NDAO will have far more applications than DAI. Interested users can give it a shot on www.DAism.io/exchange (Ropsten Test Network).
+
+## Backwards Compatibility
+[EIP-2569](https://github.com/DAism2019/EIPs/blob/master/EIPS/eip-2569.md).
+
+[EIP-2569 Pull](https://github.com/ethereum/EIPs/pull/2569).
+
+## Conclusion
```
Comment (MicahZoltu) [inline EIPS/eip-2794.md:99]: Not a template section.  Keep in mind that EIPs are standards, not papers.  This should be structured as a specification for how to do something, not a discussion of an idea.  If you just want to talk about your idea, I recommend the Ethereum research forums or a GitHub issue here.

File: EIPS/eip-2794.md [hunk 6]
```diff
@@ -0,0 +1,121 @@
...
+- Network security is heavily reliant on the price stability of ETH.
+- Beware of Derivative attacks. 
+Through our deep-dive analysis, each of these issues and risks is directly tied to ETH's prices which are affected/manipulated by trading activities in centralized crypto exchanges. In essence, if the improper staking mechanism is not redesigned these issues and risks might be triggered by ETH's price fluctuations. And our proposal completely eliminates the root cause of these issues and risks.
+
+In addition, our proposal addresses the following two points mentioned in this report and provides solutions.
+
+- While participation from Ethereum enthusiasts is important for a successful Beacon Chain launch, it is ultimately inadequate to reach sufficient levels of security. Attracting capital efficient validators will lead to higher fidelity in targeting a sufficient level of ETH staked.
+- This is a compelling serviceable addressable market, and a key objective of the incentive program to maximize network participation should be to convert these wallets into active validators.
+
+All in all, this proposal provides a solution for security and safety of ETH 2 Staking.
```
Comment (MicahZoltu) [inline EIPS/eip-2794.md:121]: Missing security section and copyright section from https://raw.githubusercontent.com/ethereum/EIPs/master/eip-template.md

File: EIPS/eip-2794.md (hunk 5 above)
Comment (zhous) [reply EIPS/eip-2794.md:99]: OK, I'll follow a template to modify the EIP.

**Pull request closed without merging**